src/components/home/lists/EditList.js

```diff
@@ -85,6 +80,7 @@ export default function EditList({ list, setModalVisible, itemListId }) {
     setModalVisible(false);
     if (itemListId != null && list != null) {
       // Delete item
+      deleteExistingItem(list);
```

Comment:
is this supposed to be receiving a `list`?

Reply:
Yes, the way I have it set up right now, lists and items use the same modal, so it was originally just for lists and now it's for items, but the item will be passed as "list". There's probably a clearer way to do that, but that's how it is for now